sdk/src/main/java/com/sygic/travel/sdk/auth/facade/AuthFacade.kt

```diff
@@ -0,0 +1,170 @@
+package com.sygic.travel.sdk.auth.facade
+
+import com.sygic.travel.sdk.Callback
+import com.sygic.travel.sdk.auth.model.AuthorizationRequest
+import com.sygic.travel.sdk.auth.model.UserRegistrationRequest
+import com.sygic.travel.sdk.auth.model.UserRegistrationResponse
+import com.sygic.travel.sdk.auth.model.UserSession
+import com.sygic.travel.sdk.auth.service.AuthService
+import com.sygic.travel.sdk.auth.service.AuthStorageService
+import com.sygic.travel.sdk.utils.runAsync
+import com.sygic.travel.sdk.utils.runWithCallback
+
+
+class AuthFacade(
+	private val authService: AuthService,
+	private val authStorageService: AuthStorageService
+) {
+
+	companion object {
+		const val CLIENT_ID = "sygictravel_android_sdk_demo"
```

Comment:
this must be configurable!

Reply:
OK, I suppose I should put clientId in`Sdk()` constructor and pass it through Kodein to `AuthService`